### ISSUE TITLE
About the size of the val dataset in paper

### ISSUE THREAD

**Issue by hyc9** [AILab-CVC/CV-VAE#13]
Thank you for your work! I would like to confirm a point regarding your paper. I noticed that you mentioned in the paper: "We evaluate our CV-VAE on the COCO2017 [18] validation dataset and the Webvid [1] validation dataset which includes 1024 videos." However, I checked the validation set of Webvid, and it contains 5000 videos. I would like to know if you randomly selected 1024 videos from that set. Thanks for your atention !

**Comment by sijeh** [comment]
Yes, we randomly selected 1024 videos from the validation set for testing.

> Thank you for your work! I would like to confirm a point regarding your paper. I noticed that you mentioned in the paper: "We evaluate our CV-VAE on the COCO2017 [18] validation dataset and the Webvid [1] validation dataset which includes 1024 videos." However, I checked the validation set of Webvid, and it contains 5000 videos. I would like to know if you randomly selected 1024 videos from that set. Thanks for your atention !
